superset-frontend/src/dashboard/components/nativeFilters/types.ts

```diff
@@ -99,11 +103,15 @@ export interface CascadeFilter extends Filter {
 export type FilterConfiguration = Filter[];
 
 export type SelectedValues = string[] | null;
+export type CurrentFilterState = {
+  value: any;
+};
```

Comment:
`CurrentFilterState` could be an alias for `JsonObject` from `@superset-ui/core` for now (making it an alias makes it easier for us to make this more specific in the future if needed).